app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: 'author_id', class_name: 'Comment'
+  has_many :posts, foreign_key: 'author_id', class_name: 'Post'
+  has_many :likes, foreign_key: 'author_id', class_name: 'Like'
```

Comment:
- [ ] OPTIONAL 
It's redundant to define `class_name` in the above associations as the `has_many` relationship already defines them. Simply do something like:
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```